src/Hero/index.js

```diff
@@ -10,13 +10,12 @@
  * governing permissions and limitations under the License.
  */
 
-/** @jsx jsx */
-import { css, jsx } from '@emotion/core'
+import React from 'react'
+import { css } from '@emotion/core'
 import { Flex } from '@react-spectrum/layout'
 import { View } from '@react-spectrum/view'
 import '@spectrum-css/typography'
 import PropTypes from 'prop-types'
-import cloneElement from '../utils'
```

Comment:
The Hero component is broken when I view it in StoryBook. I'm pretty sure you will need to add `cloneElement` back to the utils in order to apply `css` from `@emotion/core` as it relies on `jsx`.